composition-js/src/merging/reporter.ts

```diff
@@ -182,7 +182,6 @@ export class MismatchReporter {
       }
     }
     const supergraphMismatch = (supergraphElement && mismatchAccessor(supergraphElement, true)) ?? '';
-    assert(distributionMap.size > 1, () => `Should not have been called for ${supergraphElement}`);
```

Comment:
Maybe we should keep this `assert`? It make sense to assert that we have at least two things to report a mismatch here.

Reply:
Agreed with @duckki here, seems like we should keep the assert.